{{cookiecutter.project_slug}}/Procfile

```diff
@@ -1,3 +1,4 @@
+release: ./manage.py migrate
```

Comment:
Let's make it `release: manage.py migrate`, shall we? It's pretty much obvious what `manage.py` we refer to here anyway.